CMSgov/cms-carts-seds
# [CMDCT-4013] Hide Some Calculated Values in Print View

### Description

This PR shows the ability to hide certain calculated table columns and calculated values from the user in the print view of the application. Specifically, this tackles only the 2024 report’s Section 3C Part 2 and Part 3 tables and Section 3C Part 3 calculated values. If this looks like its working as intended, I’ll extend this across all of the parts and calculated values and update the 2024 base template accordingly

For a better look at what has changed, heres the normal view of Section 3C Part 2 and Part 3

Normal View:

![Screenshot 2024-09-24 at 12 38 44 PM](https://github.com/user-attachments/assets/9dce5bfd-afc7-4eb0-b9f9-df8290268c14)

![Screenshot 2024-09-24 at 12 39 14 PM](https://github.com/user-attachments/assets/01a4d6a6-6e8e-486e-b07a-788d66340013)

Print View:

![Screenshot 2024-09-24 at 12 38 56 PM](https://github.com/user-attachments/assets/cadb946b-f56b-4c00-a160-ea3c3755933e)

![Screenshot 2024-09-24 at 12 39 29 PM](https://github.com/user-attachments/assets/ab130c71-ea3e-4337-aeb0-1c0ead6302ca)


### Related ticket(s)


CMDCT-4013

---
### How to test

Open the local 2024 report as stateuser2.
Navigate to Section 3c Part 2
Note that the table shows Number and percentage
Enter some data if you want and see it calculates
Scroll down to the print button and click print section
Navigate to Part 2 and see that the number column has been hidden
Back on the normal view of the page, navigate to Navigate to Section 3c Part 3
See that theres a computed value after question 4 as well as 2 more tables
Enter data and see it works
Scroll down to the print button and click print section
Navigate to Part 3 and see that the number column has been hidden in both tables and the calculated value is also hidden.

---
### Pre-review checklist


---
### Pre-merge checklist


#### Review
